Test/Source/TestServer.cpp

```diff
@@ -438,7 +438,7 @@ static int runScript(ExternalProcess& p, const std::string& testDir, const std::
 	}
 	if (lineNum == 0)
 	{
-		std::cerr << "Invalid test script it has no lines " << testscript << std::endl;
+		//std::cerr << "Invalid test script it has no lines " << testscript << std::endl;
```

Comment:
Sounds like accidental comment-out

Reply:
Not really sure what this line intends to do but it does not compile with versions of clang >= 6.1 on compiler explorer (Ubuntu 18.04.3LTS uses 7.4) https://godbolt.org/z/XhbwkN